src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -38,6 +39,7 @@ abstract class AbstractObjectNormalizer extends AbstractNormalizer
     const SKIP_NULL_VALUES = 'skip_null_values';
     const MAX_DEPTH_HANDLER = 'max_depth_handler';
     const EXCLUDE_FROM_CACHE_KEY = 'exclude_from_cache_key';
+    const DEEP_OBJECT_TO_POPULATE = 'deep_object_to_populate';
```

Comment:
in #30888 i am adding phpdoc to these configuration constants to explain what they do and what values they accept.

Reply:
as this is already merged, i will rebase #30888 and document it there. and extract the test for this into a trait like we do the other tests there.